spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/TraceWebAutoConfiguration.java

```diff
@@ -65,6 +66,7 @@ public TraceWebAspect traceWebAspect() {
 	}
 
 	@Bean
+	@ConditionalOnMissingBean
```

Comment:
This is bad. We can't have filters with `ConditionalOnMissingBean`. If anyone has his own filter this one won't be registered